LogicalTypes.md

```diff
@@ -55,6 +57,8 @@ allows.
 implied by the `int32` and `int64` primitive types if no other annotation is
 present and should be considered optional.
 
+The sort order used for signed integer types must be `SIGNED`.
```

Comment:
My interpretation is we should fail. 
This is redundant with logical type but more explicit and deals with backward compatibility.